config/initializers/sentry.rb

```diff
@@ -1,3 +1,4 @@
 Raven.configure do |config|
   config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
+  config.transport_failure_callback = lambda { |event| Rails.logger.error(event.to_s) }
```

Comment:
why not ``` ->(event) { Rails.logger.error(event.to_s)}```
I haven't seen use use the lambda style. I have used it in the pass but was changed to the stubby version ->